app/views/idv/unavailable.html.erb

```diff
@@ -0,0 +1,36 @@
+<% title t('idv.titles.unavailable') %>
+
+<%= render AlertIconComponent.new(icon_name: :error, class: 'display-block margin-bottom-4') %>
```

Comment:
Is this a page a good candidate for `StatusPageComponent` ?

Reply:
Yes! Thank you for the suggestion. Updated in [7689c7a](https://github.com/18F/identity-idp/pull/7970/commits/7689c7a88ccd939fc51c7c913a10dce46aafc948)